### PR TITLE
fix(combobox): add text overflow for placeholder and options

### PR DESCRIPTION
**Pull request by helen-v**
## Description
<For Jira tasks:
https://jira-eng-sjc12.cisco.com/jira/browse/CAX-919
https://jira-eng-sjc12.cisco.com/jira/browse/CAX-920
>

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/[sandbox]/examples/combobox.ts [hunk 1]
```diff
@@ -8,7 +8,11 @@ const dropdownValue: string[] = [];
 
 export const comboBoxTemplate = html`
   <h3>Default</h3>
-  <md-combobox .options=${comboBoxOptions} placeholder="Placeholder" .value=${[comboBoxOptions[5]]}></md-combobox>
+  <md-combobox
+    style="width: 220px; display: inline-block;"
+    .options=${comboBoxOptions}
+    placeholder="PlaceholderPlaceholderPlaceholderPlaceholderPlaceholder"
```
Comment (yananym) [inline web-components/src/[sandbox]/examples/combobox.ts:14]: it adds truncation - but not the tooltip/title, right?

Comment (helen-v) [reply web-components/src/[sandbox]/examples/combobox.ts:14]: And title too. As I understood we need title for options, and truncate placeholder.

Comment (yananym) [reply web-components/src/[sandbox]/examples/combobox.ts:14]: Lena how come i don't see the title changes here?

Comment (helen-v) [reply web-components/src/[sandbox]/examples/combobox.ts:14]: Sorry, it's my mistake, do not merge branch with title. Added

**Review by yananym — APPROVED** [review]

**Pull request merged**